data/parameters/layout-array_parameters.yml

```diff
@@ -0,0 +1,63 @@
+epsg: 
+  len: 1
+  names: []
+  default: null
+centerLongitude: 
+  len: 1
+  unit: !astropy.units.Unit {unit: deg}
+  names: ['longitude', 'long', 'center_longitude']
+  default: null
+centerLatitude: 
+  len: 1
+  unit: !astropy.units.Unit {unit: deg}
+  names: ['latitude', 'lat', 'center_latitude']
+  default: null
+centerNorthing: 
+  len: 1
+  unit: !astropy.units.Unit {unit: m}
+  names: ['northing', 'north', center_northing']
+  default: null
+centerEasting: 
+  len: 1
+  unit: !astropy.units.Unit {unit: m}
+  names: ['easting', 'east', center_easting']
+  default: null
+centerAltitude: 
+  len: 1
+  unit: !astropy.units.Unit {unit: m}
+  names: ['altitude', 'centeralt']
+  default: null
+corsikaObsLevel: 
+  len: 1
+  unit: !astropy.units.Unit {unit: m}
+  names: ['obslevel']
+  default: null
+corsikaSphereCenter: 
+  len: 3
+  unit: [!astropy.units.Unit {unit: m}, !astropy.units.Unit {unit: m}, !astropy.units.Unit {unit: m}]
+  names: ['spherecenter']
+  default: 
+    LST: !astropy.units.Quantity
```

Comment:
Do we want to add the SCT here to have all CTA telescope types? I am a bit undecided, as it is not a telescope type for the early construction. But we have simulated it a few times and will continue to simulate it.

Reply:
For this purpose, SCT is a MST. These are the CORSIKA sphere parameters for all the telescopes named 'M-XX'.
The CORSIKA sphere center is the same for MST and SCT, right?
And the radius would be just slighlty larger for MST, so no problem.